components/style/parser.rs

```diff
@@ -81,3 +81,8 @@ impl<'a> ParserContext<'a> {
 pub fn log_css_error(input: &mut Parser, position: SourcePosition, message: &str, parsercontext: &ParserContext) {
     parsercontext.error_reporter.report_error(input, position, message);
 }
+
+/// XXXManishearth use this for everything
```

Comment:
(Maybe expand this comment, to say something like we should implement this trait for all specified value types?  I assume there's an advantage in doing so, in which case, mention that too.)